web/src/routing/routes/ForgotPassword/ForgotPassword.test.tsx

```diff
@@ -0,0 +1,90 @@
+import { MemoryRouter } from "react-router-dom";
+import { ThemeProvider } from "@features/ThemeProvider/ThemeProvider";
+import { act, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ForgotPassword from "./ForgotPassword";
+import { Auth } from "aws-amplify";
+
+const MockForgotPassword = () => (
+  <ThemeProvider>
+    <MemoryRouter>
+      <ForgotPassword />
+    </MemoryRouter>
+  </ThemeProvider>
+);
+
+describe("ForgotPassword", () => {
+  it("should show forgot password form view", () => {
+    render(<MockForgotPassword />);
+    const heading = screen.getByText("Forgot your password?");
+    const paragraph = screen.getByText(
+      "Please write email you used during create account process. We will send you a message with instructions."
+    );
+    const emailInput = screen.getByRole("textbox", { name: /Email/i });
+    const sendButton = screen.getByRole("button", {
+      name: /Send/i,
+    });
+    const goBackButton = screen.getByRole("button", {
+      name: /Go Back/i,
+    });
+
+    expect(heading).toBeInTheDocument();
```

Comment:
You don't have to check `toBeInTheDocument` afaik, as `getByText` has got already assertion included, so for example, if `Forgot your password?` text wouldn't be found, test would crash at line 19.